src/writer.js

```diff
@@ -67,7 +67,10 @@ define([
             jrm = "http://dev.commcarehq.org/jr/xforms";
         }
 
-        uuid = form.formUuid; //gets set at parse time/by UI
```

Comment:
it was mostly correct except when provided with an empty string. it goes back to major refactoring https://github.com/dimagi/Vellum/blame/master/src/parser.js#L40 so probably won't be fun trying to figure out when that happened